tool/travis.sh

```diff
@@ -59,7 +59,7 @@ elif [ "$BOT" = "test_dart2js" ]; then
     pub global activate webdev
 
     WEBDEV_RELEASE=true pub run test --reporter expanded --exclude-tags useFlutterSdk
-    WEBDEV_RELEASE=true pub run build_runner test -- --reporter expanded --exclude-tags useFlutterSdk --platform chrome-no-sandbox
+    pub run build_runner test -r -- --reporter expanded --exclude-tags useFlutterSdk --platform chrome-no-sandbox
```

Comment:
fyi @devoncarew the dart2js tests are now running with dart2js and the ddc tests are now running with ddc. that was the intended behavior, correct?
Jake believes that previously the the tests were always running with ddc. I may be missing something about how these tests were intended to run.

Reply:
They were always running with `dart2js` actually - since they were using `pub run test`, which only supports ddc.

Reply:
TBH, I think we could should remove this line entirely actually, but I will leave that up to you all :).

Reply:
so the error was more that our ddc tests in the previous section were actually running with dart2js?

Reply:
I believe that actually this only ever really meant to be relevant for the `integration_test.dart` which runs in the VM only. It spins up webdev with a `--release` arg based on the `WEBDEV_RELEASE` environment variable.

So, this was more about how devtools itself was compiled in the integration tests.

What I am changing is how the unit tests run, which was just duplicative before. Now it will run all the unit tests with both ddc and dart2js (on the respective bots).

Reply:
Yeah, just to clarify, the intent here was to run the tests against the app when it was compiled with dart2js, and to run the tests against the app when it was compiled with ddc.